Python/libraries/recognizers-date-time/recognizers_date_time/date_time/english/dateperiod_extractor_config.py

```diff
@@ -151,7 +205,42 @@ def __init__(self):
         self._range_connector_regex = RegExpUtility.get_safe_reg_exp(
             EnglishDateTime.RangeConnectorRegex)
         self._now_regex = RegExpUtility.get_safe_reg_exp(
-            EnglishDateTime.NowRegex)
+            EnglishDateTime.NowRegex
+        )
+        self._within_next_prefix_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.WithinNextPrefixRegex
+        )
+        self._time_unit_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.TimeUnitRegex
+        )
+        self._future_suffix_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.FutureSuffixRegex
+        )
+        self._ago_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.AgoRegex
+        )
+        self._later_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.LaterRegex
+        )
+        self._less_than_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.LessThanRegex
+        )
+        self._more_than_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.MoreThanRegex
+        )
+        self._duration_date_restrictions = EnglishDateTime.DurationDateRestrictions
+        self._year_period_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.YearPeriodRegex
+        )
+        self._month_num_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.MonthNumRegex
+        )
+        self._century_suffix_regex = RegExpUtility.get_safe_reg_exp(
+            EnglishDateTime.CenturySuffixRegex
+        )
+        self._ordinal_extractor = EnglishOrdinalExtractor()
+        # TODO When the implementation for these properties is added, change the None values to their respective Regexps
+        self._cardinal_extractor = None
```

Comment:
I don't get this. CardinalExtractor already exists in English. Why not used here?

Reply:
Yes, good catch. We missed it.